### DISCUSSION TITLE
Explicitly included entities should override platforms/device classes

### DISCUSSION THREAD

**Discussion by gregmac** [jseidl/magic-areas#368] [General]
I added a couple sensors explicitly, using the "Include entities to this area" option, however I noticed that they don't get included unless their device class is also in the "Presence sensors device classes" list. 

* In one case the device class (`door`) was not in the list on presence sensor device classes, yet I didn't want _all_ doors, just this one (easily solved via exclusion, but is exactly backwards from what I intended to do)
* In another, the entity didn't have a class at all

This was all a bit unexpected since I thought that list would be used only for automatic inclusion. I kind of feel like if you explicitly add an entity to the include list, that should happen regardless of the classes list.

**Comment by jseidl** [comment]
The issue is that you might want an entity to be included for aggregation of doors but not for presence sensing, thus having it bypass the selection would fix your case but mess up the other case without allowing for a solution. Keeping as is maintains both cases manageable.
